packages/dashboard/src/customization/widgets/kpi/component.css

```diff
@@ -0,0 +1,16 @@
+.kpi-widget-empty-state {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 0 2em;
+  position: relative;
+  justify-content: center;
+  text-align: center;
+}
+
+.kpi-widget-empty-state-title {
+  position: absolute;
```

Comment:
can we use some flex or grid to handle the positioning of the title? I think this will end up overlaying on the no properties or alarms text if you size the widget small enough. I think it's ok if the text overflows the widget bounds too, the other charts behave that way too.

Reply:
I was thinking maybe use SpaceBetween (which spans the whole widget) and wrap the title and the body text, and left align the title and then center align the body?